PostgreSQL.md

```diff
@@ -152,6 +170,19 @@ If backup is pushed from replication slave, WAL-G will control timeline of the s
 
 ``backup-push`` can also be run with the ``--permanent`` flag, which will mark the backup as permanent and prevent it from being removed when running ``delete``.
 
+#### Rating composer mode
+
+In the rating composer mode, WAL-G places files with similar updates frequencies in the same tarballs during backup creation. This is the recommended option to use in pair with `backup-fetch` [redundant archives skipping](#redundant-archives-skipping). Be aware that although rating composer allows saving more data, it may result in slower backup creation compared to the default tarball composer.
```

Comment:
-> recommended
Remove it

Reply:
Removed recommended https://github.com/wal-g/wal-g/pull/756/commits/9653426e4bd92774d502b99f2b224d6ad3d21954